slides/bitcoin-primer.tex

```diff
@@ -0,0 +1,216 @@
+% Choose one to switch betweeen slides and handout
+\documentclass[]{beamer}
+%\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Bitcoin Primer}
+\author{Prof. Dr. Fabian Schär}
+\institute{University of Basel}
+
+% Config File
+\input{../config/config.tex}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{document}
+
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+
+%%%
+\begin{frame}{Definition}
+Some key aspects of Bitcoin: \vspace{1em}
+
+	\begin{enumerate}
+		\item<2-> Competitive creation
+		\item<3-> Virtual representation
+		\item<4-> Decentralized management
+	\end{enumerate}
+	\vspace{1em}	
+\uncover<5->{Bitcoin is a decentralized data structure.} \\	\vspace{1em}
+\uncover<6->{The system is maintained by its participants and works in the absence of centralized third parties.} 
+\end{frame}
+%%%	
+
+%%%
+\begin{frame}{Bitcoin Building Blocks}
+	\begin{figure}[h!]
+		\center
+		\input{../assets/figures/bitcoin-system.tex}
+		\label{fig:bitcoinsystem}
+	\end{figure}
+
+\begin{center}
+\uncover<2->{
+Because of the decentralized infrastructure, the transaction \textbf{capacity}, transaction \textbf{legitimacy} and transaction \textbf{consensus} are much more difficult to achieve.
+}
+\end{center}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction Capacity}
+\textbf{Goal:} ensuring the ability to initiate a transaction  \\
+\vspace{1em}
+\uncover<2->{Bitcoin network 
+        \begin{itemize}
+		\item {consists of all users}
+		\item {primary communication path}
+		%\item<2->{completely decentral}
+		\end{itemize}
+		}
+\vspace{1em}
+\uncover<3->{Bitcoin protocol
+        \begin{itemize}
+        \item<3->{defines manner in which communication has to take place (ruleset)}
+        \end{itemize}
+        }     
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction Capacity}  %example banking vs. decentral system?
+	
+	\begin{figure}[h!]
+		\center
+		\input{../assets/figures/transaction-capacity.tex}
+	\end{figure}\vspace{1em}
+
+\uncover<2->{       
+	Peer-to-Peer Network: 
+        \begin{itemize}
+		\item<1-> permissionless
+		\item<1-> censorship-resistant
+		\item<1-> No special privileges
+		\end{itemize}
+		}
+\end{frame}
```

Comment:
I suggest that we remove the bullet points and reduce the amount of text for the first trx-capacity slide. Nonetheless, these things are important and should be said during the video. 

```suggestion
\begin{frame}{Transaction Capacity}
\textbf{Goal:} ensuring the ability to initiate a transaction  \\
	\begin{figure}[h!]
		\center
		\input{../assets/figures/transaction-capacity.tex}
	\end{figure}\vspace{1em}

\uncover<2->{       
	Peer-to-Peer Network: 
        \begin{itemize}
		\item<1-> Permissionless
		\item<1-> Censorship-resistant
		\item<1-> No special privileges
		\end{itemize}
		}
\end{frame}
```
Alternatively, use one more uncover command:
1. Goal line
2. Figure
3. P2P network summary

I really like the following slides about transaction legitimacy and consensus!